src/SixLabors.Fonts/Tables/AdvancedTypographic/Shapers/DefaultShaper.cs

```diff
@@ -128,6 +128,50 @@ public override void AssignFeatures(IGlyphShapingCollection collection, int inde
                 this.AddFeature(collection, index, count, VertTag);
             }
 
+            // User defined fractional features require special treatment.
+            // https://docs.microsoft.com/en-us/typography/opentype/spec/features_fj#tag-frac
+            if (this.HasFractions())
+            {
+                this.AssignFractionalFeatures(collection, index, count);
+            }
+
+            // Add user defined features.
```

Comment:
Maybe make this an extra method name instead of a comment?

Reply:
Och it's just a for-loop. I wouldn't bother.